api/src/sign_unsign_zip.rs

```diff
@@ -25,7 +25,7 @@ where
     let mut input = ZipArchive::new(BufReader::new(input)).map_err(CopyZipError::InputZip)?;
     let mut output = ZipWriter::new(BufWriter::new(output));
 
-    output.set_raw_comment(input.comment().to_owned());
+    output.set_raw_comment(input.comment().to_owned().into());
```

Comment:
Thank you for the PR! Have you tried if the `.to_owned()` is still needed, or if the `into()` alone might be enough?

Reply:
You're right, no need for `.to_owned()` anymore